Sources/RInAppMessaging/Views/Presenters/BaseViewPresenter.swift

```diff
@@ -86,7 +80,7 @@ extension BaseViewPresenter {
         }
         // Broadcast only `impression` type here. Other types are sent after campaign is closed.
         let impressionData = [impression].encodeForAnalytics()
-        AnalyticsBroadcaster.sendEventName(Constants.RAnalytics.impressionsEventName,
+        AnalyticsTracker.sendEventName(Constants.RAnalytics.impressionsEventName,
                                            dataObject: [Constants.RAnalytics.Keys.impressions: impressionData,
```

Comment:
nit: please fix indentation in `dataObject` lines

Reply:
Fixed